tokens/v-breadcrumb.yml

```diff
@@ -0,0 +1,28 @@
+#  ==========================================================================
+# breadcrumb.yml
+#
+#   TOC:
+#
+#  Base
+#  Size Variants
+#    @MD-LG
+#    @XS-SM
+#  ==========================================================================
+props:
+  breadcrumb-text-color:
+    value: "{!taken-for-granite}"
+    type: color
+  breadcrumb-last-child-text-color:
+    value: "{!heart-of-darkness}"
+    type: color
+  breadcrumb-hover-text-color:
+    value: "{!heart-of-darkness}"
+    type: color
+  breadcrumb-delimiter-text-color:
+    value: "{!heart-of-darkness}"
+    type: color
+global:
+  category: breadcrumb
+  type: token
```

Comment:
this could be color rather than token and then you would not need to pass a type to each token as they all seem to be color anyhow

Reply:
Ok, I lack context of the taxonomy logic.  I'll change it.